DependencyInjection/CmfSeoExtension.php

```diff
@@ -250,20 +251,34 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
     {
         $loader->load('sitemap.xml');
 
-        $container->setParameter($this->getAlias().'.sitemap.configurations', $config['configurations']);
-
+        $configurations = $config['configurations'];
         foreach ($config['configurations'] as $key => $configuration) {
-            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+//            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
```

Comment:
thought that is the chance to create a default sitemap equal to the manager definition in the doctrine bundles.